lib/octokit/client/source_import.rb

```diff
@@ -41,6 +41,27 @@ def source_import_progress(repo, options = {})
         get "#{Repository.path repo}/import", options
       end
 
+      # Update source import with authentication/project_choice
+      # Restart source import if no options are passed
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository.
+      # @return [Sawyer::Resource] Hash representing the repository import
+      # @see https://developer.github.com/v3/migration/source_imports/#update-existing-import
+      # @option options [String] :vcs_username If authentication is required, the username to provide to vcs_url.
+      # @option options [String] :vcs_password If authentication is required, the password to provide to vcs_url.
+      # @option options [String] :vcs The originating VCS type. Can be one of "subversion", "git", "mercurial", or "tfvc".
+      # @option options [String] :tfvc_project For a tfvc import, the name of the project that is being imported.
```

Comment:
Instead of listing vcs and tfvcproject, can this refer to the project choices list on the get endpoint? The exact attributes should be one of the project choices that the API returned.